### ISSUE TITLE
raise Exception does not trigger refactoring analysis and a custom exception does

### ISSUE THREAD

**Issue by oserban** [sourceryai/public#5]
### Issue description or question

```python
from common.errors import ValidationError


def set_status(status: str) -> str:
    if status:
        if status.upper().strip() == "ON":
            result = "starting"
        elif status.upper().strip() == "OFF":
            result = ""
        else:
            # raise Exception() does not trigger any refactor
            raise ValidationError(title="Invalid status", description="Status can be on or off")
    else:
        raise ValidationError(title="Invalid status", description="Status can be on or off")

    return result
```

### Code editor or IDE name and version

Pycharm 2019.1.3

### OS name and version

Linux

**Comment by brendanator** [comment]
What would you expect this example to be refactored to?

**Comment by oserban** [comment]
It's not about the expected refactoring, but the fact that the analysis does not trigger if you change the type of the exception. It's inconsistent. The refactoring output is similar to issue #6

**Comment by brendanator** [comment]
In issue #6 the refactoring removes the duplicated line
```python
    raise ValidationError(title="Invalid status", description="Status can be on or off")
``` 

In this example though there is no duplication to remove, so no refactoring is suggested

**Comment by oserban** [comment]
```python
def set_status(status: str) -> str:
    if status:
        if status.upper().strip() == "ON":
            result = "starting"
        elif status.upper().strip() == "OFF":
            result = ""
        else:
            raise Exception()
    else:
        raise Exception()

    return result
```

should trigger a remove duplicate code as well, but for some reason it does not

**Comment by brendanator** [comment]
Ah ha! Thanks

**Comment by brendanator** [comment]
Interestingly, I expect this will be solved by fixing #6

**Comment by Hellebore** [comment]
Looks like this should indeed be fixed by fix for #6, which is under review.

**Comment by Hellebore** [comment]
Fix now deployed.
